cli/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use std::str::FromStr;
+
+use clap::{command, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct CliArguments {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    /// Read the JSON string file in input_file_path and parse it into a VerifiableCredential. Then prints the VerifiableCredential
+    TestIssueCredential {
+        #[arg(short, long)]
+        input_file: String,
+    },
+    /// Read the JSON string file in input_file_path and parse it into a VerifiablePresentation. Then prints the VerifiablePresentation
```

Comment:
Change the comment here (and the one above) to `input_file` instead of `input_file_path`